[keyboard]Graystudio BD60 Support (#22210)
qmk/qmk_firmware: keyboards/gray_studio/bd60/readme.md

Reviewer: Could an Imgur hosted image of this PCB be added please?

Author: @dunk2k Here is the link https://imgur.com/a/WvNsuSQ

# Gray Studio bd60

 A 60% keyboard by Graystudio. PCB designed and manufactured by DEMO Studio.

 * Keyboard Maintainer: [edwardslau](https://github.com/edwardslau)
 * Hardware Support: bd60

>  * Hardware Availability:

Reviewer 2: Fill in or remove
```suggestion
```

 Make example for this keyboard (after setting up your build environment):

>  	make gray_studio bd60:default
> 
>  Flashing example for this keyboard:
> 
>  	make gray_studio bd60:default:flash

Reviewer 2: ```suggestion
 	make gray_studio/bd60:default

 Flashing example for this keyboard:

 	make gray_studio/bd60:default:flash
```

> ## Bootloader
> 
> * **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
> 
> See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more infromation. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

Reviewer 2: ```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

# Gray Studio bd60

> 

Reviewer: PCB image
```suggestion

![BD60](https://i.imgur.com/BTneFkMh.jpeg)

```

> # Gray Studio bd60

Reviewer: ```suggestion
# Gray Studio BD60
```

![BD60](https://i.imgur.com/BTneFkMh.jpeg)

 A 60% keyboard by Graystudio. PCB designed and manufactured by DEMO Studio.

 * Keyboard Maintainer: [edwardslau](https://github.com/edwardslau)

>  * Hardware Support: bd60

Reviewer: ```suggestion
 * Hardware Support: BD60 PCB
```

 Make example for this keyboard (after setting up your build environment):

>  	make gray_studio/bd60:default
> 
>  Flashing example for this keyboard:
> 
>  	make gray_studio/bd60:default:flash

Reviewer 3: ```suggestion
    make gray_studio/bd60:default

 Flashing example for this keyboard:

    make gray_studio/bd60:default:flash
```
